### ISSUE TITLE
Steam Cloud can't be disabled

### ISSUE THREAD

**Issue by whew-inc** [ValveSoftware/steam-for-linux#7905]
#### Your system information
```
Operating System: openSUSE Tumbleweed 20210629
KDE Plasma Version: 5.22.2
KDE Frameworks Version: 5.83.0
Qt Version: 5.15.2
Kernel Version: 5.13.0_tkg_bmq-2.gfa5dcf1-default (64-bit)
Graphics Platform: Wayland
Processors: 16 × AMD Ryzen 7 2700X Eight-Core Processor
Memory: 15.5 GiB of RAM
Graphics Processor: AMD Radeon RX 5700 XT
```

* Steam client version (build number or date): `Jun 8 2021 22:23:41`
* Distribution (e.g. Ubuntu): `OpenSUSE Tumbleweed`
* Opted into Steam client beta?: `No`
* Have you checked for system updates?: `Yes`

#### Please describe your issue in as much detail as possible:
Even after disabling Steam Cloud sync for a game AND globally, Steam still tries to sync to cloud.
This can waste tons of time, especially if the game has a huge save file (ex. Noita). I'm talking about **minutes** before Steam finally launches a game. In Noita's case, running the game's binary directly with Wine circumvents this. Sadly this also means no Steam integration.
KSysGuard does not report increased network activity from Steam, so I'm confused.

#### Steps for reproducing this issue:
1. Disable Steam Cloud globally and/or for a specific game
2. Launch said game

Expectation: Steam prepares to launch the game, then the game launches
Reality: Steam prepares to launch the game, after which it synchronizes with cloud... THEN it launches.

This has been going on for a while time. Feels like it's always been there really

**Comment by kisak-valve** [comment]
Hello @whew-inc, this issue is already being tracked at #7518. Closing as a duplicate.

**Comment by serossi** [comment]
> Hello @whew-inc, this issue is already being tracked at #7518. Closing as a duplicate.

are you for real, you closed #7518 with reference on this report too. so both bug reports are closed because the other exist.
meanwhile the issue still persist, 3 years later

**Comment by kisak-valve** [comment]
Hello @serossi, no, as of this writing #7518 is an open issue report as indicated in the top left corner of that issue report.

**Comment by serossi** [comment]
> Hello @serossi, no, as of this writing #7518 is an open issue report as indicated in the top left corner of that issue report.

it is not, it is closed with reference to #7905 BY YOU
